### ISSUE TITLE
Errors when running examples

### ISSUE THREAD

**Issue by DamienIrving** [pyviz/geoviews#334]
I'm new to geoviews, so the first thing I did was try and run some of the examples. 
This involved installing the package,
```
$ conda install -c pyviz geoviews
```
and downloading the examples:
```
$ conda install pyc
$ geoviews examples 
```
I then tried the following two example jupyter notebooks but got errors:

### world_population.ipynb

```
import pandas as pd
import geoviews as gv

gv.extension('bokeh')

# Define data
cities = pd.read_csv('../../assets/cities.csv', encoding="ISO-8859-1")
population = gv.Dataset(cities, kdims=['City', 'Country', 'Year'])
points = population.to(gv.Points, ['Longitude', 'Latitude'], ['Population', 'City', 'Country'])

tiles = gv.tile_sources.Wikipedia

# Plot
%%opts Points (size=0.003 cmap='viridis') [tools=['hover'] size_index=2 color_index=2 width=600]
tiles * points
```
Warnings:
```
WARNING:param.OverlayPlot05720: No plotting class for WMTS type and bokeh backend found. 
WARNING:param.OverlayPlot05720: No plotting class for Points type and bokeh backend found. 
bokeh backend could not plot any Elements in the Overlay.
```
Output:
```
:HoloMap   [Year]
   :Overlay
      .WMTS.I   :WMTS   [Longitude,Latitude]
      .Points.I :Points   [Longitude,Latitude]   (Population,City,Country)
```

Were those warnings fatal (i.e. is that why the image didn't appear)? Or have I missed a command required to make bokeh plots appear in a notebook?

### city_populations_2050.ipynb

```
import pandas as pd
import geoviews as gv
import geoviews.tile_sources as gvts

gv.extension('bokeh')

# Declare data
cities = pd.read_csv('../../assets/cities.csv', encoding="ISO-8859-1")

points = gv.Points(cities, ['Longitude', 'Latitude'], ['City', 'Population', 'Country', 'Year'],
                   label='Top 20 Cities by population in 2050').select(Year=2050).sort('Population').iloc[-20:]

# Plot
(gvts.CartoMidnight *
 points.options(
    width=700, height=600, size_index=3, color_index=3, cmap='viridis',
    global_extent=True, size=0.01, tools=['hover'], show_legend=False
 ) *
 gv.Labels(points).options(
    text_font_size='8pt', text_color='white'
 )
)
```
Warnings:
```
WARNING:param.main: Option 'global_extent' for Points type not valid for selected backend ('bokeh'). Option only applies to following backends: ['matplotlib']
WARNING:param.OverlayPlot03119: No plotting class for WMTS type and bokeh backend found. 
WARNING:param.OverlayPlot03119: No plotting class for Points type and bokeh backend found. 
WARNING:param.OverlayPlot03119: No plotting class for Labels type and bokeh backend found. 
bokeh backend could not plot any Elements in the Overlay.
```
Output:
```
:Overlay
   .WMTS.I                                     :WMTS   [Longitude,Latitude]
   .Points.Top_20_Cities_by_population_in_2050 :Points   [Longitude,Latitude]   (City,Population,Country,Year)
   .Labels.Top_20_Cities_by_population_in_2050 :Labels   [Longitude,Latitude]   (City,Population,Country,Year)
```
Again, were those warnings fatal (i.e. is that why the image didn't appear)? Or have I missed a command required to make bokeh plots appear in a notebook?

**Comment by DamienIrving** [comment]
The corresponding world population example also didn't work because:
```
from geoviews import dim
```
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-b7737fd84143> in <module>
----> 1 from geoviews import dim

ImportError: cannot import name 'dim' from 'geoviews' (/anaconda3/lib/python3.7/site-packages/geoviews/__init__.py)
```

**Comment by philippjfr** [comment]
Which version of holoviews and geoviews did you get? Looks like old versions.

**Comment by philippjfr** [comment]
Obviously still an issue since you installed straight from pyviz and should have the latest versions.

**Comment by DamienIrving** [comment]
@philippjfr From `$ conda list`:
```
holoviews                 1.11.0             pyh39e3cac_0    pyviz
geoviews                  1.5.0                      py_0    conda-forge
```

**Comment by philippjfr** [comment]
Could you try explicitly pinning holoviews 1.12.3 and geoviews 1.6.2 when conda installing?

**Comment by DamienIrving** [comment]
That was an easy fix! Upgrading geoviews alone worked:
```
$ conda install -c pyviz geoviews=1.6.2

geoviews                  1.6.2                      py_0    pyviz
holoviews                 1.11.0             pyh39e3cac_0    pyviz
```
as did going the extra step and upgrading holoviews too:
```
conda install -c pyviz holoviews=1.12.3
```

Thanks so much for your help, @philippjfr.

**Comment by rchak007** [comment]
> That was an easy fix! Upgrading geoviews alone worked:
> 
> ```
> $ conda install -c pyviz geoviews=1.6.2
> 
> geoviews                  1.6.2                      py_0    pyviz
> holoviews                 1.11.0             pyh39e3cac_0    pyviz
> ```
> 
> as did going the extra step and upgrading holoviews too:
> 
> ```
> conda install -c pyviz holoviews=1.12.3
> ```
> 
> Thanks so much for your help, @philippjfr.

did not work for me. I will update later if it works.

**Comment by dwr-psandhu** [comment]
This happens now if python is left unpinned ( python 3.12 gets installed) and that forces geoviews to resolve to 1.5.0
If i pin python<3.12 then geoviews resolves to 1.10.0

**Comment by Hoxbro** [comment]
I will update GeoViews to Bokeh 3.3 and Python 3.12, but it will likely first happen next week.
